content/agent/changelog.md

```diff
@@ -1,13 +1,12 @@
 ---
 title: "Changelog"
```

Comment:
I think we need a different name, maybe

```suggestion
title: "Agent changelog"
```

Otherwise, we have two changlogs, same name, diferent purposes, next to each other

<img width="273" alt="Screenshot 2025-06-09 at 2 23 03 PM" src="https://github.com/user-attachments/assets/ebd4e870-e1ef-452e-b278-7e1526584d03" />